odoo/addons/base/models/res_config.py

```diff
@@ -535,7 +535,7 @@ def default_get(self, fields):
                         _logger.warning(WARNING_MESSAGE, value, field, icp)
                         value = 0.0
                 elif field.type == 'boolean':
-                    value = value.lower() == 'true'
+                    value = ustr(value).lower() == 'true'
```

Comment:
what is it for?

Reply:
that's basically a bug fixing. A bug was exposed as we <a href="https://github.com/odoo/odoo/pull/22158/files?diff=unified#diff-4886ada66586ac25a10d879ce2af725aR61"/>added</a> a `Boolean` field in `res.config.settings` model with `default` and `config_parameter` attributes together. Slight more details available in commit message of this bug fixing, https://github.com/odoo/odoo/pull/22158/commits/98d1931886f8160be927ddb05f8d898563b0cc63